app/routes.py

```diff
@@ -1,2 +1,190 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from sqlalchemy import desc, asc
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
 
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+
+@task_bp.route("", methods=["POST"])
+def handle_tasks():
+    """This endpoint takes in a json object and creates a new Task instance"""
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body\
+            or "completed_at" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
+    new_task = Task(
+        title=request_body["title"],
+        description=request_body["description"],
+        completed_at=request_body["completed_at"]
+    )
+    db.session.add(new_task)
+    db.session.commit()
+    return jsonify({"task": new_task.task_dict()}), 201
+
+
+@task_bp.route("", methods=["GET"])
+def get_tasks():
+    """ This endpoint takes in a GET request and possibly a sorting query and returns 
+    a sorted list of task instances or a non-sorted list of task instances """
+    sort_query = request.args.get("sort")
+    if sort_query == "desc":
+        tasks = Task.query.order_by(desc("title"))
+        task_list = [task.task_dict() for task in tasks]
+    elif sort_query == "asc":
+        tasks = Task.query.order_by(asc("title"))
+        task_list = [task.task_dict() for task in tasks]
+    else:
+        tasks = Task.query.all()
+        task_list = [task.task_dict() for task in tasks]
+    return jsonify(task_list), 200
```

Comment:
We don't need to repeat the `task_list` list comprehension logic within every conditional branch, as `tasks` will always be created regardless.

```suggestion
    if sort_query == "desc":
        tasks = Task.query.order_by(desc("title"))
    elif sort_query == "asc":
        tasks = Task.query.order_by(asc("title"))
    else:
        tasks = Task.query.all()
    task_list = [task.task_dict() for task in tasks]
    return jsonify(task_list), 200
```